### ISSUE TITLE
Branching strategy

### ISSUE THREAD

**Issue by dstoft** [jakobhviid/ProjectLeapNode#1]
We have already discussed the branching strategy a bit, but I think that we should document it abit, to make sure that we are picturing the same thing.

**Comment by dstoft** [comment]
See changes in branch: https://github.com/jakobhviid/ProjectLeapNode/tree/feature/issue-1-branching-strategy

**Comment by mads5606** [comment]
Agreed, if nothing else, then also just to ease it if someone else needs to be introduced to the project at some point.
